### PR TITLE
Feature+Fix: two Issues in character generation and homeland feature

### PR DESCRIPTION
**Pull request by sebastiansIT**
1. Add i18n-Labels to generation of notes for generated characters (Childhood and Formative Events)
2. Change all datasets for char gen. Consumables are now index based instead of Dice-Based. This is necessary because the systems values for consumables are defined as 1=D6, 2=D8, 3=D10 and 4=D12. The old values 6,8,10,12 doesn't work in the actual Version of the System.
3. Add the optional random selection of a homeland region and write it to the generated note. I adopted this from the German version of "Legends & Adventurers"

### CODE REVIEW

**Comment by aMediocreDad** [comment]
I don't see the actual homelands added in the German dataset. Is this intentional?

File: src/components/character-generator/character-converter.js [hunk 1]
```diff
@@ -152,11 +152,16 @@ export class CharacterConverter {
...
+			: "";
 		const childhood = character.childhood;
-		let notes = `<h3>Childhood: ${childhood.name}</h3><p>${childhood.description}</p>`;
+		let notes = `<h3>${game.i18n.localize("FLCG.CHILDHOOD")}: ${childhood.name}</h3>
+			<p>${childhood.description}</p>
+			${homeland}`;
 		for (let i = 0; i < character.formativeEvents.length; i++) {
 			const event = character.formativeEvents[i];
-			notes += `<h3>Formative Event: ${event.name}</h3><p>${event.description}</p>`;
+			notes += `<h3>${game.i18n.localize("FLCG.EVENT")}: ${event.name}</h3><p>${event.description}</p>`;
```
Comment (aMediocreDad) [inline src/components/character-generator/character-converter.js:164]: Thank you for finding and removing hardcoded language strings.

File: static/assets/datasets/chargen/dataset-de.json
Comment (aMediocreDad) [inline static/assets/datasets/chargen/dataset-de.json:1]: Big thanks for this too!

File: static/lang/en.json [hunk 2]
```diff
@@ -161,6 +161,7 @@
 	"FLCG.ATTRIBUTES": "Attributes",
 	"FLCG.CHILDHOOD": "Childhood",
 	"FLCG.EVENT": "Formative Event",
+	"FLCG.HOMELAND": "You comes from",
```
Comment (aMediocreDad) [inline static/lang/en.json:164]:
```suggestion
	"FLCG.HOMELAND": "You hail from",
```

**Review by aMediocreDad — CHANGES_REQUESTED** [review]
Small change for English localization string. I am also interested in knowing how this homeland thing works, seeing as it is only in the German-language version?

**Comment by sebastiansIT** [comment]
> I don't see the actual homelands added in the German dataset. Is this intentional?

I have a personal version for bitter reach and added it to this data set. For the base game I only have a german printed version of Legends & Adventurers but I can transfer it to the german default dataset if you want.

It ist only an Array of Strings for each kin like this: "homeland": ["Northfall", "Northfall", "Silver Coast", "Silver Coast", "Silver Coast", "Raven Land"],

**Comment by aMediocreDad** [comment]
> For the base game I only have a german printed version of Legends & Adventurers but I can transfer it to the german default dataset if you want.

That'd be great!

**Comment by aMediocreDad** [comment]
Also, just to be clear. This issue #331 is not fixed in this PR?

**Comment by sebastiansIT** [comment]
> Also, just to be clear. This issue #331 is not fixed in this PR?

No, it isn't. There are two possible solutions and I don't want to decide which one is the best.

**Comment by sebastiansIT** [comment]
> > For the base game I only have a german printed version of Legends & Adventurers but I can transfer it to the german default dataset if you want.
> 
> That'd be great!

Done! But your Pull Request Liniting is failed for this commit.

**Review by aMediocreDad — APPROVED** [review]

**Pull request merged**